app/views/layouts/_footer.html.erb

```diff
@@ -1,4 +1,3 @@
-
-    <div class="footer">
-      <p>Powered by <a href="http://www.agileventures.org">AgileVentures</a> volunteers &#9734; Open Source software</p>
-    </div>
+<div class="footer">
+  <p>Powered by <a href="http://www.agileventures.org">AgileVentures</a> volunteers &#9734; Open Source software</p>
+</div>
```

Comment:
Given that this is not a public-facing site, could the footer provide different info (e.g. link to the G+ support community) or be dispensed with altogether?